docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -0,0 +1,333 @@
+---
+product: date-pickers
+title: Components lifecycle
+githubLabel: 'component: pickers'
+packageName: '@mui/x-date-pickers'
+---
+
+# Components lifecycle
+
+<p class="description">This page explains when the callbacks onChange, onAccept and onClose are called.</p>
+
+## Fields lifecycle
+
+### On simple fields
+
+:::info
+The information below are applicable on standalone fields (when rendering `<DateField />`),
+but also on pickers for the field editing (when rendering `<DatePicker />` and using the input to edit the value).
+:::
+
+The field components have an internal state to update the visible value.
+
+It will only call the `onChange` callback when:
+
+- the user fills one section of an empty input. The value equals `Invalid date`.
+- the user completes all sections of an input. The value reflects the input.
+- the user cleans one section of a completed input. The value equals `Invalid date`.
+- the user cleans all sections of an input. The value equals `null`.
+
+In the example below, `onChange` will be called when any of the conditions are triggered:
+
+{{"demo": "LifeCycleDateFieldEmpty.js", "defaultCodeOpen": false}}
+
+### On range fields [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+On range fields (`SingleInputDateRangeField` / `MultiInputDateRangeField` / ... ),
+`onChange` will be called if the date you are modifying is matching one of the condition above,
+even if the other date does not.
+
+In the example below, changing the value of the start date section will call `onChange` even if the end date is empty or partially filled.
+
+{{"demo": "LifeCycleDateRangeField.js", "defaultCodeOpen": false}}
+
+## Pickers lifecycle
+
+### When is `onClose` called ?
+
+:::info
+If you are not controlling the `open` prop, then all the scenarios below also describe when the picker closes.
```

Comment:
> **also** describe when the picker closes 

I am not 100% sure I understand this 🤔 The title here is 'pickers lifecycle' and this section describes when `onClose` is called. Does it describe something else? Maybe the use of the word 'also' here makes the phrasing a bit confusing.

Reply:
Let's centralize the discussion around this `info` in https://github.com/mui/mui-x/pull/8372#discussion_r1229863435 :+1: